docs/install/from_source.rst

```diff
@@ -22,24 +22,23 @@ To install alpa from source, we need to build these forks.
 
 2. Install dependencies
 
-  - CUDA Toolkit: cuda and cudnn
+  - CUDA Toolkit: `cuda <https://developer.nvidia.com/cuda-toolkit>`_, `cudnn <https://developer.nvidia.com/cudnn>`_, and `nccl <https://developer.nvidia.com/nccl>`_
```

Comment:
I met 'Could not find any nccl.h matching version '2.09.9' in any subdirectory' error. Should we specify the nccl version to install?